docs/guides/button-upgrade-guide.md

```diff
@@ -8,23 +8,29 @@ order: 4
-    - [Upgrade examples for variant=”link” and has "padding overrides plus href"](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-variant-link-or-link-inverse-upgrade-examples-for-link-variant-with-an-href-attribute-and-padding-overrides)
-    - [Upgrade examples for variant="link" and has "padding overrides and no href"](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-variant-link-or-link-inverse-upgrade-examples-for-link-variant-with-no-href-attribute-and-padding-overrides)
-    - [Upgrade examples for variant="link" and has "no padding overrides and no href"](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-variant-link-or-link-inverse-upgrade-example-for-link-variant-with-no-href-attribute-and-no-padding-overrides)
-  - [Upgrading __CloseButton__](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-closebutton)
-    - [Upgrade examples for __CloseButton__](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-closebutton-upgrade-examples-for-closebutton)
+- [Upgrading **icon only** buttons, no visible text](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-variant-icon,-icon-inverse,-circle-default,-circle-primary,-or-circle-danger)
 
+  - [Upgrade examples for "icon" or "icon-inverse"](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-variant-icon,-icon-inverse,-circle-default,-circle-primary,-or-circle-danger-upgrade-examples-for-icon-or-icon-inverse)
+  - [Upgrade examples for "circle-default", "circle-primary", or "circle-danger"](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-variant-icon,-icon-inverse,-circle-default,-circle-primary,-or-circle-danger-upgrade-examples-for-circle-default,-circle-primary,-or-circle-danger)
+
+- [Upgrading **link variant** values](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-variant-link-or-link-inverse)
+  - [Upgrade examples for variant=”link” and has "padding overrides plus href"](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-variant-link-or-link-inverse-upgrade-examples-for-link-variant-with-an-href-attribute-and-padding-overrides)
+  - [Upgrade examples for variant="link" and has "padding overrides and no href"](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-variant-link-or-link-inverse-upgrade-examples-for-link-variant-with-no-href-attribute-and-padding-overrides)
+  - [Upgrade examples for variant="link" and has "no padding overrides and no href"](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-variant-link-or-link-inverse-upgrade-example-for-link-variant-with-no-href-attribute-and-no-padding-overrides)
+- [Upgrading **CloseButton**](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-closebutton)
+  - [Upgrade examples for **CloseButton**](#button-upgrade-guide/#button-upgrade-for-version-8.0-upgrading-closebutton-upgrade-examples-for-closebutton)
+
+##### Note about Codemods
```

Comment:
I would make it even more highlighted, probably even with a code snippet that shows that it needs to bed imported from the latest version of InstUI even if you are on v7, etc.

Reply:
added more explanation here.